src/main.cpp

```diff
@@ -0,0 +1,11 @@
+#include <iostream>
+#include <emu.h>
+
+int main()
+{
+	Emu::Instr instr;
+	word_t opcode = 0x15c0;
```

Comment:
`// 0x15c0 = 0012700 = (see PDP1145, p.27, p.66) = MOV (PC)+, R0 = load PC value into R0 and increment PC`
And why would you use hex format?